### PR TITLE
Init release scripts

### PR DESCRIPTION
**Pull request by cramja**
Includes:
- command to create artifacts
- command to publish release candidates
- command to test a release candidate

All commands are parameterized by the .profile file

@hakanmemisoglu is it perfect?

### CODE REVIEW

File: release/release_manager.profile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# folders corresponding to SVN repos where we keep our release artifacts
+export SVN_DEV=svn-quickstep-dev
+export SVN_REL=svn-quickstep-rel
+
+# you probably will not need to change things below this line
+export PROJECT_NAME=apache-quickstep-incubating
+
+# path to apache repo with quickstep releases
+export SVN_DEV_URL=https://dist.apache.org/repos/dist/dev/incubator/quickstep
+export SVN_REL_URL=https://dist.apache.org/repos/dist/rel/incubator/quickstep
```
Comment (zuyu) [inline release/release_manager.profile]: Add `\n` here.

File: release/release_cmds.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+  # first download third_party, then do your normal build
+  cd third_party/
+  ./download_and_patch_prerequisites.sh
+  cd ..
+  cd build
+  cmake ..
+  make
+}
+
```
Comment (zuyu) [inline release/release_cmds.sh]: Remove this empty line.

**Comment by cramja** [comment]
@jianqiao it's gtg

**Comment by jianqiao** [comment]
LGTM! Merging.

**Pull request merged**